ui/components/component-library/select-wrapper/select-wrapper.types.ts

```diff
@@ -0,0 +1,66 @@
+import type {
+  StyleUtilityProps,
+  PolymorphicComponentPropWithRef,
+} from '../box';
+import { PopoverStyleUtilityProps } from '../popover';
+
+export interface SelectWrapperStyleUtilityProps extends StyleUtilityProps {
+  /*
+   * Additional classNames to be added to the SelectWrapper component
+   */
+  className?: string;
+  /*
+   * Children of SelectWrapper component are put inside the popover
```

Comment:
At first I thought this was a mistake but it makes sense to me I wonder if we could make it a little clearer? Not a great suggestion
```suggestion
   * Children of SelectWrapper component are put inside the select menu which uses the Popover component
```